_sass/_base.scss

```diff
@@ -4,23 +4,65 @@
 body, h1, h2, h3, h4, h5, h6,
 p, blockquote, pre, hr,
 dl, dd, ol, ul, figure {
-    margin: 0;
-    padding: 0;
+  margin: 0;
+  padding: 0;
 }
 
 
 
 /**
+ * Variables
+ */
+$navy-blue-color: rgb(22,33,63);
+$navy-blue-mid-color: rgb(30,46,80);
+$navy-blue-light-color: rgb(13,56,96);
+$dark-red-color: #ad0a18;
+$base-font-family: 'Lato';
+$base-font-size: 16;
+$small-font-size: 8;
+$background-color: #ffffff;
+$brand-color: #0d3960;
+$grey-color: #808080;
+$grey-color-light: #cccccc;
+$text-color: #000000;
+$spacing-unit: 1;
+$base-line-height: 1;
+$content-width: 1280;
+
+$tablet-min-width: 768px;
+$small-desktop-min-width: 992px;
+$large-desktop-min-width: 1200px;
+
+@mixin tablet {
+  @media (min-width: #{$tablet-min-width}) and (max-width: #{$small-desktop-min-width - 1px}) {
+    @content;
+  }
+}
+
+@mixin small-desktop {
+  @media (min-width: #{$small-desktop-min-width}) and (max-width: #{$large-desktop-min-width - 1px}) {
+    @content;
+  }
+}
+
+@mixin large-desktop {
+  @media (min-width: #{$large-desktop-min-width}) {
+    @content;
+  }
+}
+
+
+/**
  * Basic styling
  */
 body {
-    font-family: $base-font-family;
-    font-size: $base-font-size;
-    line-height: $base-line-height;
-    font-weight: 300;
-    color: $text-color;
-    background-color: $background-color;
-    -webkit-text-size-adjust: 100%;
+  font-family: $base-font-family;
```

Comment:
Properties should be ordered background-color, color, font-family, font-size, font-weight, line-height, -webkit-text-size-adjust